matrix/Dcm.hpp

```diff
@@ -20,6 +20,12 @@ class Quaternion;
 template<typename Type>
 class Euler;
 
+/**
+ * Direction cosine matrix class
+ *
+ * The rotation between two coordinate frames is
```

Comment:
Maybe we should say that for uav applications frame 1 is earth and frame 2 is body. The Matrix library only supports one axis angle sequence, and the point is that this makes it clean and you can easily tranform between dcm, euler, and quaternions without worrying about it. I don't think we should ever support more than one euler sequence.